src/Discord.Net.Rest/Entities/Messages/RestUserMessage.cs

```diff
@@ -136,7 +136,7 @@ public Task RemoveReactionAsync(IEmote emote, IUser user, RequestOptions options
             => MessageHelper.RemoveReactionAsync(this, user, emote, Discord, options);
         public Task RemoveAllReactionsAsync(RequestOptions options = null)
             => MessageHelper.RemoveAllReactionsAsync(this, Discord, options);
-        public Task<IReadOnlyCollection<IUser>> GetReactionUsersAsync(IEmote emote, int limit = 100, ulong? afterUserId = null, RequestOptions options = null)
+        public IAsyncEnumerable<IReadOnlyCollection<IUser>> GetReactionUsersAsync(IEmote emote, int limit = 100, ulong? afterUserId = null, RequestOptions options = null)
```

Comment:
Make limit required and remove afterUserId

Reply:
It's handled as part of the async enumerable returned, so it's unnecessary.

Reply:
How does the parameter get passed to MessageHelper.GetReactionUsersAsync then?

Reply:
It doesn't need to be; our async enumerable code handles it automatically.

Reply:
Ohhhh, I see what you mean, where do I get the starting point for the Paged Enumerable then or should I leave that out?

Reply:
The paged enumerable defaults to 0, which would retrieve all of the results anyway.